corehq/apps/accounting/bootstrap/features.py

```diff
@@ -1,31 +1,39 @@
+"""
+These are the feature allocations for all tiers of our Software Plan Editions
+available for self-service.
+"""
 from corehq import privileges
 
 paused_v0 = []
 
 # COMMUNITY PLANS
 
+# Grandfathered Community Plans created prior to August 2018
 community_v0 = [
     privileges.PROJECT_ACCESS,
-    privileges.CASE_SHARING_GROUPS,
-    privileges.CHILD_CASES,
     privileges.EXCEL_DASHBOARD,
     privileges.DAILY_SAVED_EXPORT,
+    privileges.CASE_SHARING_GROUPS,
+    privileges.CHILD_CASES,
 ]
 
+# Grandfathered Community Plans created prior to Dec 18, 2019
 community_v1 = [
     privileges.PROJECT_ACCESS,
     privileges.CASE_SHARING_GROUPS,
     privileges.CHILD_CASES,
 ]
 
-# V2 created in December 2019
+# Current Community Plan
 community_v2 = [
```

Comment:
nit: this comment will become outdated and wrong if it is not removed or updated when a new plan is added. The previous comment seemed better. Maybe the variable name could be changed to include the date so the comment could be removed entirely? Something like `community_2019`

The same applies to similar comments below.

Reply:
I think the potential issue with this is that an ops user in 2020 might think that `community_2019` is suddenly outdated. That's why I feel keeping the versioning makes a little more sense.

Reply:
and to clarify, these role slugs are visible in the billing UI.